provisioning_templates/PXEGrub2/pxegrub2_default_local_boot.erb

```diff
@@ -4,7 +4,7 @@ name: PXEGrub2 default local boot
 %>
 <%# Used to boot provisioned hosts, do not associate or change the name. %>
 
-set default=0
+set default=<%= global_setting("default_pxe_item_local", "local_chain_hd0") %>
```

Comment:
Shouldn't we use the `index` here as well?

Reply:
Scratch that: I'm reviewing top-bottom, I see the changes in the snippets now :)